nl.lproj/Localizable.strings

```diff
@@ -0,0 +1,160 @@
+"MENU_NEW_STATION" = "Zender…";
+"MENU_NEW_GROUP" = "Verzameling…";
+
+"DISCOVERABILITY_HUD_NEW_STATION" = "Nieuwe zender";
+"DISCOVERABILITY_HUD_NEW_GROUP" = "Nieuwe verzameling";
+
+"MENU_VIEW_AS_GRID" = "als raster";
+"MENU_VIEW_AS_LIST" = "als lijst";
+
+"DISCOVERABILITY_HUD_VIEW_AS_GRID" = "Bekijk als raster";
+"DISCOVERABILITY_HUD_VIEW_AS_LIST" = "Bekijk als lijst";
+
+"MENU_VIEW_SHOW_SIDEBAR" = "Toon navigatiekolom";
+"MENU_VIEW_HIDE_SIDEBAR" = "Verberg navigatiekolom";
+
+"MENU_HELP_GETSUPPORT" = "Krijg ondersteuning op Twitter…";
+"MENU_HELP_SHOW_WELCOME_SCREEN" = "Toon welkomstpaneel";
+
+"PLAYBACK_ERROR_STALL" = "De verbinding met de zender is verbroken.";
+
+"SEARCH_PLACEHOLDER" = "Zoek";
+
+"ALERT_OK" = "OK";
+"ALERT_ADD" = "Voeg toe";
+"ALERT_CANCEL" = "Annuleer";
+"ALERT_DELETE" = "Verwijder";
+
+"ADD_STATION_WINDOW_TITLE" = "Voeg nieuwe zender toe";
+"EDIT_STATION_WINDOW_TITLE" = "Wijzig zender";
+
+"EDIT_STATION_WINDOW_NAME_PLACEHOLDER" = "Naam";
+"EDIT_STATION_WINDOW_URL_PLACEHOLDER" = "http://pad/naar/stream";
+"EDIT_STATION_WINDOW_IMAGE_ACCESSIBILITY_LABEL" = "Kies afbeelding";
+
+"ADD_COLLECTION_WINDOW_TITLE" = "Voeg nieuwe verzameling toe";
+"EDIT_COLLECTION_WINDOW_TITLE" = "Wijzig verzameling";
+
+"EDIT_COLLECTION_WINDOW_NAME_PLACEHOLDER" = "Mijn verzameling";
+
+"SIDEBAR_CONTEXT_EDIT_COLLECTION" = "Wijzig";
+"SIDEBAR_CONTEXT_DELETE_COLLECTION" = "Verwijder verzameling";
+"SIDEBAR_CONTEXT_SHARE_COLLECTION" = "Deel";
+
+"STATION_CONTEXT_EDIT_COLLECTION" = "Wijzig";
+"STATION_CONTEXT_DELETE_COLLECTION" = "Verwijder zender";
+"STATION_CONTEXT_MOVE" = "Verplaats naar Verzameling";
+
+"CONTEXT_ALERT_DELETE_COLLECTION_TITLE" = "Verwijder \"%@\"";
+"CONTEXT_ALERT_DELETE_COLLECTION_BODY" = "Bent u zeker dat u deze collectie en alle zenders erin wenst te verwijderen?";
+
+"CONTEXT_ALERT_DELETE_STATION_TITLE" = "Verwijder \"%@\"";
+"CONTEXT_ALERT_DELETE_STATION_BODY" = "Bent u zeker dat u deze zender wenst te verwijderen?";
+
+"COLLECTION_VIEW_EMPTY_SECTION_TITLE" = "Geen zenders";
+
+"WELCOME_COPY" = "Welkom bij Broadcasts! Als u wenst, kunnen we enkele voorkeurszenders van bij ons toevoegen waarmee u uw bibliotheek kan starten. Of, als u dit verkiest, kunt u handmatig uw eigen stations toevoegen of kies uit duizenden stations van de lijst.";
+
+"WELCOME_UPSELL" = "Welkom bij Broadcasts! In deze gratis versie kunnen we enkele voorkeurszenderes van bij ons toevoegen waarmee u uw bibliotheek kan starten. Of, als u liever upgradet naar de betaalde versie, kunt u uw eigen stations handmatig toevoegen of kiezen uit duizenden zenders van de lijst.";
+
+"WELCOME_ADD_PRESETS" = "Voeg voorkeuren toe";
+"WELCOME_NO_THANKS" = "Nee, bedankt";
+
+"GENERATED_COLLECTION_TITLE" = "Mijn verzameling";
+
+"UPSELL_COPY" = "Bedankt om Broadcasts te gebruiken!\n\nAls u dit wilt, kunt u de geavanceerde functies van Broadcasts ontgrendelen met een in-app-aankoop.\n\nOf, als u dit wilt, kunt u de vooraf ingestelde zenders blijven gebruiken en bewerken!";
```

Comment:
"Bedankt voor het gebruiken van Broadcast!\n\nAls u dit wilt, kunt u de geavanceerde functies van Broadcasts ontgrendelen met een in-app-aankoop.\n\nOf, als u dit wilt, kunt u de vooraf ingestelde zenders blijven gebruiken en bewerken!"